src/main/java/org/docx4j/openpackaging/parts/WordprocessingML/FootnotesPart.java

```diff
@@ -72,6 +79,22 @@ public void init() {
 		
 	}
 
+	/**
+     * Convenience method to getJaxbElement().getEndnote()
+     * @since 2.8.1
+     */
+    public List<Object> getContent() {
```

Comment:
By the look of it, this won't work properly.  getContent().add(foo) doesn't add foo to the endnote object

Reply:
I've tried to let FootnotePart and EndnotePart implement the same ContentAccessor as FooterPart and HeaderPart, to handle each part in a similar way. In some previous version, the getContent() method has been un-commented. Is it possible to let FootnotePart and EnddnotePart implement ContentAccessor in a proper way?

If it's not possible to let FootnotePart and EndnotePart implement ContentAccessor, this particullar change can be reverted.  The most important part, is the ability to update content controls bindings in these parts.